99robots-custom-image-sizes.php

```diff
@@ -1,14 +1,14 @@
 <?php
 /**
  * Plugin Name:	Custom Image Sizes by 99 Robots
- * Plugin URI:	https://draftpress.com/products/?utm_source=plugin-uri&utm_medium=custom-image-sizes&utm_campaign=plugins-page
+ * Plugin URI:	https://draftpress.com/custom-image-sizes/
  * Description:	Custom Image Sizes by 99 Robots is a quick and simple way for you to add your own image sizes to your WordPress site.
- * Version: 1.2.6
- * Author: 99 Robots
- * Author URI: https://draftpress.com
+ * Version: 1.2.7
+ * Author: DraftPress
+ * Author URI: https://draftpress.com/
  * License: GPL2
  * License URI:	http://www.gnu.org/licenses/gpl-2.0.txt
- * Text Domain:	99robots-custom-image-sizes
+ * Text Domain:	draftpress-custom-image-sizes
```

Comment:
can't change the text domain just like that..if you're changing it here..then it needs to be changed at all places inside of code..either revert this back or change it at all places..

Reply:
you can read more about it here: https://developer.wordpress.org/themes/functionality/internationalization/#text-domain